### PR TITLE
move mesos address to mesos option section only

### PR DESCRIPTION
**Pull request by chlgit**

### CODE REVIEW

File: dev/config/MASTER.yaml [hunk 1]
```diff
@@ -1,15 +1,46 @@
...
       actions:
         - name: "first"
-          command: "echo hi"
+          command: "/bin/sleep 5m"
+          executor: mesos
+          docker_image: docker-paasta.yelpcorp.com:443/services-robj-test-service:paasta-e9fc795ec881cfd1b9d4b68540504690deddd4ec
+          cpus: 1
+          mem: 100
+          extra_volumes:
+              - container_path: '/nail/etc/mrjob'
```
Comment (qui) [inline dev/config/MASTER.yaml]: Can we take this out too?

File: tron/config/schema.py [hunk 2]
```diff
@@ -110,6 +110,7 @@ def config_object_factory(name, required=None, optional=None):
 ConfigMesos = config_object_factory(
     name='ConfigMesos',
     optional=[
+        'master_address',
```
Comment (qui) [inline tron/config/schema.py]: OK, can we either 1) make this required or 2) add validation that this exists if enabled is True?

File: tron/mesos.py [hunk 3]
```diff
@@ -38,6 +38,7 @@ class MesosClusterRepository:
     """A class that stores MesosCluster objects and configuration."""
 
     clusters = {}
+    mesos_master_address = None,
```
Comment (qui) [inline tron/mesos.py]: nit: don't need these commas

File: tron/config/schema.py [hunk 4]
```diff
@@ -109,6 +109,9 @@ def config_object_factory(name, required=None, optional=None):
 
 ConfigMesos = config_object_factory(
     name='ConfigMesos',
+    required=[
+        'master_address',
```
Comment (solarkennedy) [inline tron/config/schema.py]: Does this mean you will have to add `mesos_address` to all our master files when we upgrade?

Comment (chlgit) [reply tron/config/schema.py]: Yes. I will update the master files later.

**Pull request merged**